mpv.el

```diff
@@ -362,5 +362,14 @@ of \\[universal-argument] will add another `mpv-seek-step' seconds."
   (interactive)
   (mpv--enqueue '("revert-seek") #'ignore))
 
+;;;###autoload
+(defun mpv-version ()
+  "Return mpv version."
```

Comment:
Maybe also explain the behavior on interactive use here? Something like the following, perhaps?
```suggestion
  "Return the mpv version string.
When called interactively, also show a more verbose version in
the echo area."
```